src/actions/index.ts

```diff
@@ -13,10 +13,14 @@ import {
   LOGIN,
   LOGIN_ERROR,
   LOGOUT,
+  DISSONANT_VOICES_LOGIN_STARTED,
+  DISSONANT_VOICES_LOGIN,
+  DISSONANT_VOICES_LOGIN_ERROR,
+  DISSONANT_VOICES_LOGOUT,
 } from './types';
 import { AppState } from '@reducers';
 
-import { getAccessToken, signInFlow, signOutFlow } from '@lib/auth';
+import { getAccessToken, signInFlow, signOutFlow } from '@lib/dissonantVoices';
```

Comment:
Hmm, does this replace the old login/logout functions?  

Seems like maybe these actions should go in their own file (src/audio/actions.ts or something like that).